rhel.Dockerfile

```diff
@@ -0,0 +1,129 @@
+
+# NOTE: uncomment for local build. Must also set full registry path in FROM to registry.redhat.io or registry.access.redhat.com
+# enable rhel 7 or 8 content sets (from Brew) to resolve jq as rpm
+COPY content_sets_epel7.repo /etc/yum.repos.d/
+
+RUN microdnf install -y findutils bash wget yum gzip tar jq python3-six python3-pip && microdnf -y clean all && \
+    # install yq (depends on jq and pyyaml - if jq and pyyaml not already installed, this will try to compile it)
+    if [[ -f /tmp/root-local.tgz ]] || [[ ${BOOTSTRAP} == "true" ]]; then \
+      mkdir -p /root/.local; tar xf /tmp/root-local.tgz -C /root/.local/; rm -fr /tmp/root-local.tgz;  \
+      /usr/bin/pip3.6 install --user yq jsonschema; \
+      # could be installed in /opt/app-root/src/.local/bin or /root/.local/bin
+      for d in /opt/app-root/src/.local /root/.local; do \
+        if [[ -d ${d} ]]; then \
+          cp ${d}/bin/yq ${d}/bin/jsonschema /usr/local/bin/; \
+          pushd ${d}/lib/python3.6/site-packages/ >/dev/null; \
+            cp -r PyYAML* xmltodict* yaml* yq* jsonschema* /usr/lib/python3.6/site-packages/; \
+          popd >/dev/null; \
+        fi; \
+      done; \
+      chmod -c +x /usr/local/bin/*; \
+    else \
+      /usr/bin/pip3.6 install yq jsonschema; \
+    fi && \
+    ln -s /usr/bin/python3.6 /usr/bin/python && \
+    # test install worked
+    for d in python yq jq jsonschema; do echo -n "$d: "; $d --version; done
+
+# for debugging only
+# RUN microdnf install -y util-linux && whereis python pip jq yq && python --version && jq --version && yq --version
+
+################# 
+# PHASE TWO: configure registry image
+#################
+
+COPY ./scripts/*.sh ./scripts/meta.yaml.schema /build/
+COPY /v3 /build/v3
+WORKDIR /build/
+
+# if only including the /latest/ plugins, apply this line to remove them from builder
+# RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/" | grep -o ".*/")
+
+# not supported in Brew unless we prefetch the content via tarball injection
+# optional steps for air gap - replace references to docker.io, quay.io, registry.access.redhat.com, registry.redhat.io with internal registry
+# RUN ./list_containers.sh v3 && ./replace_container_repos.sh v3 myquay.mycorp.com
+
+RUN ./check_plugins_location.sh v3 && \
+    ./set_plugin_dates.sh v3 && \
+    ./check_plugins_viewer_mandatory_fields.sh v3 && \
+    ./ensure_latest_exists.sh && \
+    ./index.sh v3 > /build/v3/plugins/index.json && \
+    chmod -c -R g+rwX /build
+
+################# 
+# PHASE THREE: create ubi8-minimal image with httpd
+################# 
+
+# Build registry, copying meta.yamls and index.json from builder
+# https://access.redhat.com/containers/?tab=tags#/registry.access.redhat.com/rhscl/httpd-24-rhel7
+FROM registry.access.redhat.com/rhscl/httpd-24-rhel7:2.4-104 AS registry
+USER 0
+
+# BEGIN these steps might not be required
+RUN sed -i /etc/httpd/conf/httpd.conf \
+    -e "s,Listen 80,Listen 8080," \
```

Comment:
`s,Listen 80,Listen 8080,` is not required as the default is `Listen 0.0.0.0:8080` (so there aren't matches), but the other lines do make changes (still don't know if they're necessary).